<?xml version="1.0"?>
<launch>
  <!-- Launch a map from the maps folder-->
  <!-- <arg name="map" default="$(find f1tenth_gym_ros)/maps/berlin.yaml"/> -->
  <arg name="map" default="$(env RACE_MAP_PATH)"/>
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map)"/>

  <!-- Launch robot model -->
  <include file="$(find f1tenth_gym_ros)/launch/racecar_model.launch"/>

  <!-- Launch gym bridge node -->
  <rosparam command="load" file="$(find f1tenth_gym_ros)/params.yaml"/>
  <param name="map_path" value="$(arg map)"/>
  <node pkg="f1tenth_gym_ros" name="gym_bridge" type="gym_bridge_bare.py" output="screen" required="true"/>
</launch>
